packages/assets-controllers/CHANGELOG.md

```diff
@@ -13,6 +13,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - **BREAKING:** Bump peer dependency `@metamask/phishing-controller` to `^13.0.0` ([#6098](https://github.com/MetaMask/core/pull/6098))
 
+### Fixed
```

Comment:
Nit: can we move this CHANGELOG under the `Unreleased` section instead of version `71.0.0`